### DISCUSSION TITLE
Simple script that returns my Public IP address

### DISCUSSION THREAD

**Discussion by securegh** [Ulauncher/Ulauncher#1207] [Q&A]
Hello, 

I am looking for a simple script that returns the value of 
`curl ifconfig.co/`

Thanks!

**Comment by friday** [comment]
That's not hard to find, unless you specifically want it to use ifconfig.co. Use https://ext.ulauncher.io/ or https://www.google.com/search?q=ulauncher+my+ip+extension

**Reply by securegh** [reply]
I searched on Google and looked at the extensions available, all of them need an to talk to an API, that's not what am looking for. 
Instead of an extension, can the Script option in shortcut do this function `curl ifconfig.co`?
